pyproject.toml

```diff
@@ -37,11 +37,13 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9,<3.11"
-
+python = "^3.9,<3.13"
+Pillow = "10.1.0"
```

Comment:
**suggestion:** first merge release branch to dev, to have Pillow "~10.3.0"

Reply:
or merge can happen after, but probably good to have it now and make sure tests are passing fine with newer python versions